src/taco-cli/cli/run.ts

```diff
@@ -167,21 +161,31 @@ class Run extends commands.TacoCommandBase {
         });
     }
 
+    private runLocal(localPlatforms?: string[]): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
+        if (this.data.options["livereload"] || this.data.options["devicesync"]) {
+            // intentionally delay-requiring it since liveReload fetches whole bunch of stuff
+            return require("./liveReload").startLiveReload(!!this.data.options["livereload"], !!this.data.options["devicesync"], localPlatforms);
```

Comment:
Ooh wait, scratch that.
Just looking at this line: https://github.com/Microsoft/TACO/blob/b1de83577c878bf81774ba34389514f8efc656a1/src/taco-cli/cli/liveReload.ts#L33,

I realize that we use Cordova.tryInvoke(...)